ci/licenses_golden/licenses_flutter

```diff
@@ -1425,8 +1425,8 @@ FILE: ../../../flutter/shell/platform/fuchsia/flutter/flutter_runner_product_con
 FILE: ../../../flutter/shell/platform/fuchsia/flutter/focus_delegate.cc
 FILE: ../../../flutter/shell/platform/fuchsia/flutter/focus_delegate.h
 FILE: ../../../flutter/shell/platform/fuchsia/flutter/focus_delegate_unittests.cc
-FILE: ../../../flutter/shell/platform/fuchsia/flutter/fuchsia_external_view_embedder.cc
-FILE: ../../../flutter/shell/platform/fuchsia/flutter/fuchsia_external_view_embedder.h
+FILE: ../../../flutter/shell/platform/fuchsia/flutter/gfx_external_view_embedder.cc
```

Comment:
This needs to be sorted alphabetically, thats why the license check is failing

Reply:
Done.